pom.xml

```diff
@@ -120,8 +128,34 @@
 		</dependency>
 	</dependencies>
 
+	<repositories>
+		<!-- Sonatype repository from which to obtain SNAPSHOT dependencies created by Reactome (or others sources) -->
+		<repository>
+			<id>oss.sonatype.org-snapshot</id>
+			<url>http://oss.sonatype.org/content/repositories/snapshots</url>
+			<releases>
+				<enabled>false</enabled>
+			</releases>
+			<snapshots>
+				<enabled>true</enabled>
+			</snapshots>
+		</repository>
+	</repositories>
+
 	<build>
 		<plugins>
+			<!-- sets the Java compiler version targeted by Maven -->
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-compiler-plugin</artifactId>
+				<version>3.8.1</version>
+				<configuration>
+					<source>1.8</source>
+					<target>1.8</target>
```

Comment:
Oh, I _just_ realized you're doing this _everywhere_ now...

Reply:
Yeah, I removed the maven compiler properties in favour of stating it explicitly in the compiler plugin in the all the projects.

Reply:
Yeah, I thought you were only tweaking one project.

Reply:
I just thought it would be good to be consistent since I'm going over all the projects anyways :).

Reply:
Yeah... I like the other style. But whatever... :man_shrugging: ;)